boards/nxp/imx95_evk_15x15/Kconfig.imx95_evk_15x15

```diff
@@ -0,0 +1,6 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_IMX95_EVK_15X15
```

Comment:
Why do you need to add a new board for 15x15?
On a quick look all files you added here are the same as those from imx95_evk.

Reply:
if revisions are awkward because it's more of a parallel design than sequential, you can still make common files to share between the boards, instead of duplicating everything